tests/unit/gaze/gaze_transform_test.py

```diff
@@ -791,6 +791,82 @@ def fixture_experiment():
             id='clip',
         ),
 
+        pytest.param(
+            {
+                'data': pl.from_dict(
+                    {
+                        'time': [1000, 1000],
+                        'x_pix': [(100 - 1) / 2 + 10, (100 - 1) / 2 + 10],
+                        'y_pix': [10.0, 10.0],
+                        'distance': [1000, 1000],
+                    },
+                ),
+                'experiment': pm.Experiment(
+                    sampling_rate=1000,
+                    screen_width_px=100,
+                    screen_height_px=100,
+                    screen_width_cm=100,
+                    screen_height_cm=100,
+                    distance_cm=None,
+                    origin='center',
+                ),
+                'pixel_columns': ['x_pix', 'y_pix'],
+                'distance_column': 'distance',
+            },
+            'pix2deg', {'center_offset': (10, 10)},
```

Comment:
here it would be nice if `center_offset` would be included in the `Screen` class in `Experiment`, just like it is with `origin`. This way one would only need to define the center offset once in the Definition and further computations will be taken care of.

Would you be willing to include this in this PR? Depending on your motivation this can also be worked on in a follow-up PR, as this will lead to small changes further down the road.